### PR TITLE
add java queries

### PR DESCRIPTION
**Pull request by andy-bell101**
Small PR to add Java queries.

The plugin seems to work in the actual editor, but when I tried to write test cases I couldn't get it to work for some reason. Inspecting it with the `screen:snapshot_util()` function would show up everything except the class declaration. Not sure what was going on there. Happy to include a test case if someone can advise.

### CODE REVIEW

**Comment by lewis6991** [comment]
It would be great if you could just include some example text in `test/`. I can do all the tests later at some point.

File: queries/java/context.scm [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  body: (_) @context.end
+) @context
+
+(class_declaration
+  body: (_) @context.end
+) @context
+
+([
+  (expression_statement)
+] @context)
```
Comment (lewis6991) [inline queries/java/context.scm:23]:
```suggestion
(expression_statement) @context
```

**Review by lewis6991 — APPROVED** [review]

File: queries/java/context.scm (hunk 1 above)
Comment (andy-bell101) [reply queries/java/context.scm]: Oops didn't notice this. Fixed it myself

**Comment by daedroza** [comment]
Can support for kotlin be added as well?

**Pull request merged**